src/sentry/static/sentry/app/views/organizationDiscover/conditions/utils.jsx

```diff
@@ -101,7 +101,7 @@ export function getExternal(internal, columns) {
 
     if (type === 'datetime') {
       const date = moment.utc(external[2]);
-      external[2] = date.isValid() ? date.format() : null;
+      external[2] = date.isValid() ? date.format('YYYY-MM-DDTHH:mm:ss') : null;
```

Comment:
What does snuba/clickhouse do when it gets invalid data? Does it 500 or 40x?

Reply:
500